test/studies/shootout/pidigits/thomasvandoren/pidigits-BigInt.suppressif

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env perl
```

Comment:
Personally I would write this in bash instead of perl, just because I think we're trending towards trying to use python or bash. That said, I just added a perl suppressif since I copied it from another test so it's not a big deal either way. 

The first comment line gets turned into a description of the suppressif so I think the comment should be made a one liner. I think you can also simplify the logic to be `CHPL_TEST_PERF== on and CHPL_TARGET_COMPILER==cray-prgenv-cray` instead of the nested if

Here's what I would do (but I don't mind if you do the equivalent in perl): 

```bash
#!/usr/bin/env bash

# Believed to be CCE bugs (related to -hipa), see JIRA issue 64

if [ "$CHPL_TEST_PERF" = "on" ] && [ "$CHPL_TARGET_COMPILER" = "cray-prgenv-cray" ]
then
    echo "True"
else
    echo "False"
fi
```